_episodes/l3-03-colaborative-development.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Does it make more sense for this filename to be `l2-03...` rather than `l3-03...` now the course is being split into two sessions?